src/server/pps/server/worker_rc.go

```diff
@@ -635,6 +628,12 @@ func (a *apiServer) getWorkerOptions(pipelineInfo *pps.PipelineInfo) (*workerOpt
 	}, nil
 }
 
+func makeWorkerPachctlSecretName(pipelineInfo *pps.PipelineInfo) string {
+	x := "pachctl-secret-" + pipelineInfo.Pipeline.Name
```

Comment:
still need to handle underscores, should resolve some of the test failures

also maybe make the "localhost:1653" below based on the env

Reply:
I'm not sure what you mean. Do you mean use the env ValueFrom to get it from kubernetes?  Or should we set it explicitly when we create the worker, and then look there?

The localhost is something we can depend on.  Pods share a networking namespace.  Whatever creates this secret will be doing it with knowledge of how the pod is set up.  Maybe moving it to a constant would be a good idea.  I think trying to get it from k8s is going to be more brittle that this, but I'm not sure exactly what you had in mind.

Reply:
oh sorry, I just meant the port. `PeerPort` is configurable